code for tinkering audio/tone_generation.py

```diff
@@ -0,0 +1,14 @@
+#function for generating a tone.
+
+def toneGeneration(sec, freq, amplitude):
+    buildSin = makeEmptySoundBySeconds(secs)                  #building the clip
+    sampleRate = getSamplingRate(buildSin)
```

Comment:
[Naming Conventions](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables). Should be: sample_rate.